examples/mujoco/td3/train_td3.py

```diff
@@ -0,0 +1,201 @@
+        # Cast observations to float32 because our model uses float32
+        env = chainerrl.wrappers.CastObservationToFloat32(env)
+        if args.monitor:
+            env = gym.wrappers.Monitor(env, args.outdir)
+        if args.render and not test:
+            env = chainerrl.wrappers.Render(env)
+        return env
+
+    env = make_env(test=False)
+    timestep_limit = env.spec.tags.get(
+        'wrapper_config.TimeLimit.max_episode_steps')
+    obs_space = env.observation_space
+    action_space = env.action_space
+    print('Observation space:', obs_space)
+    print('Action space:', action_space)
+
+    action_size = action_space.low.size
+
+    winit = chainer.initializers.LeCunUniform(3 ** -0.5)
+
+    policy = chainer.Sequential(
+        L.Linear(None, 400, initialW=winit),
+        F.relu,
+        L.Linear(None, 300, initialW=winit),
+        F.relu,
+        L.Linear(None, action_size, initialW=winit),
+        F.tanh,
+        chainerrl.distribution.ContinuousDeterministicDistribution,
+    )
+    policy_optimizer = optimizers.Adam().setup(policy)
+
+    def make_q_func_with_optimizer():
+        q_func = chainer.Sequential(
+            concat_obs_and_action,
+            L.Linear(None, 400, initialW=winit),
+            F.relu,
+            L.Linear(None, 300, initialW=winit),
+            F.relu,
+            L.Linear(None, 1, initialW=winit),
+        )
+        q_func_optimizer = optimizers.Adam().setup(q_func)
+        return q_func, q_func_optimizer
+
+    q_func1, q_func1_optimizer = make_q_func_with_optimizer()
+    q_func2, q_func2_optimizer = make_q_func_with_optimizer()
+
+    # Draw the computational graph and save it in the output directory.
+    fake_obs = chainer.Variable(
+        policy.xp.zeros_like(obs_space.low, dtype=np.float32)[None],
+        name='observation')
+    fake_action = chainer.Variable(
+        policy.xp.zeros_like(action_space.low, dtype=np.float32)[None],
+        name='action')
+    chainerrl.misc.draw_computational_graph(
+        [policy(fake_obs)], os.path.join(args.outdir, 'policy'))
+    chainerrl.misc.draw_computational_graph(
+        [q_func1(fake_obs, fake_action)], os.path.join(args.outdir, 'q_func1'))
+    chainerrl.misc.draw_computational_graph(
+        [q_func2(fake_obs, fake_action)], os.path.join(args.outdir, 'q_func2'))
+
+    rbuf = replay_buffer.ReplayBuffer(10 ** 6)
+
+    explorer = explorers.AdditiveGaussian(
+        scale=0.1, low=action_space.low, high=action_space.high)
+
+    def burnin_action_func():
+        """Select random actions until model is updated one or more times."""
+        return np.random.uniform(
+            action_space.low, action_space.high).astype(np.float32)
+
+    # Hyperparameters in http://arxiv.org/abs/1802.09477
+    agent = chainerrl.agents.TD3(
+        policy,
+        q_func1,
+        q_func2,
+        policy_optimizer,
+        q_func1_optimizer,
+        q_func2_optimizer,
+        rbuf,
+        gamma=0.99,
```

Comment:
It would not help replicating the paper, because it would not change the behavior, right? Can you elaborate on how would it help?

Reply:
It would not help in replicating the paper, but it would improve clarity, so that somebody using this as a reference knows it's a hyperparameter of importance.

Reply:
I see, sure it may help to show that it is an important hyperprameter.